test/parallel/test-worker-memory.js

```diff
@@ -47,5 +47,7 @@ for (let i = 0; i < numWorkers; ++i) {
                 'Unexpected memory overhead: ' +
                 util.inspect([startStats, finishStats]));
     }
+
+    assert.strictEqual(process.memoryUsage().rss, process.rss());
```

Comment:
Leftover from the previous iteration of the PR? There's no `process.rss()` anymore.